homeassistant/components/tasmota/strings.json

```diff
@@ -0,0 +1,18 @@
+{
+  "config": {
+    "step": {
+      "config": {
+        "description": "Please enter the Tasmot configuration.",
+        "data": {
+          "discovery_prefix": "Discovery topic"
+        }
+      }
+    },
+    "abort": {
+      "single_instance_allowed": "Only a single configuration of Tasmota is allowed."
```

Comment:
Please use as many references as possible. See eg the adguard integration strings.